spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -1,29 +1,49 @@
-== Kafka Streams Binding Capabilities of Spring Cloud Stream
+== Usage
 
-Spring Cloud Stream Kafka support also includes a binder specifically designed for Apache Kafka Streams binding.
-Using this binder, applications can be written that leverage the Apache Kafka Streams API.
-For more information on Kafka Streams, see https://kafka.apache.org/documentation/streams/developer-guide[Kafka Streams API Developer Manual]
+For using the Kafka Streams binder, you just need to add it to your Spring Cloud Stream application, using the following 
+Maven coordinates:
 
-Kafka Streams support in Spring Cloud Stream is based on the foundations provided by the Spring Kafka project.
-For details on that support, see http://docs.spring.io/spring-kafka/reference/html/_reference.html#kafka-streams[Kafaka Streams Support in Spring Kafka].
+[source,xml]
+----
+<dependency>
+  <groupId>org.springframework.cloud</groupId>
+  <artifactId>spring-cloud-stream-binder-kafka-streams</artifactId>
+</dependency>
+----
 
-Here are the maven coordinates for the Spring Cloud Stream Kafka Streams binder artifact.
+Alternatively, you can also use the Spring Cloud Stream RabbitMQ Starter.
```

Comment:
Why the mention of RabbitMQ starter here?

Reply:
Good catch, I was reusing the text from Rabbit binder, but I forgot to change it. 🤦‍♂️ - I'll fix it.